Documentation/spelling_wordlist.txt

```diff
@@ -585,6 +585,7 @@ sandboxing
 scalable
 sctl
 seccomp
+seccompProfile
```

Comment:
@ishuar with the removal of the initial defaults in the `SecurityContext`, this entry in the spelling wordlist is no longer necessary

Reply:
@mhofstetter  removed